pkg/envoy/testdata/fake-envoy

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
 # This script pretends to be Envoy in unit tests. It captures the flags and the
-# bootstrap config from the named pipe specified via `--config-path`, and writes
-# them to the file at `--test-output` (which is read and checked in the test).
+# bootstrap config specified via `--config-yaml`, and writes them to the file at
+# `--test-output` (which is read and checked in the test).
 # It then sleeps for 10 minutes to check we're correctly killing the process.
 
 set -e
 
-config_path=""
+config_yaml=""
```

Comment:
This fake envoy process will just stash the value of the `--config-yaml` argument it receives and write it to a file later so that the calling test can make assertions on it. The changes in this test are just around handling `--config-yaml` instead of `--config-path`.